sdk/cosmos/azure-cosmos/src/samples/java/com/azure/cosmos/AsyncContainerCodeSnippets.java

```diff
@@ -279,4 +280,105 @@ public void queryItemsAsyncSqlQuerySpecSample() {
             .subscribe();
         // END: com.azure.cosmos.CosmosAsyncContainer.SqlQuerySpec.queryItems
     }
+
+    public void databaseReadAsyncSample() {
+        // BEGIN: com.azure.cosmos.CosmosAsyncDatabase.read
+        CosmosAsyncDatabase database = cosmosAsyncClient
+            .getDatabase("<YOUR DATABASE NAME>");
+        database.read().subscribe(databaseResponse -> {
+                System.out.println(databaseResponse);
+            },
+            throwable -> {
+                throwable.printStackTrace();
+            });
+        // END: com.azure.cosmos.CosmosAsyncDatabase.read
+    }
+
+    public void databaseDeleteAsyncSample() {
+        // BEGIN: com.azure.cosmos.CosmosAsyncDatabase.delete
+        CosmosAsyncDatabase database = cosmosAsyncClient
+            .getDatabase("<YOUR DATABASE NAME>");
+        database.delete().subscribe(databaseResponse -> {
+                System.out.println(databaseResponse);
+            },
+            throwable -> {
+                throwable.printStackTrace();
+            });
+        // END: com.azure.cosmos.CosmosAsyncDatabase.delete
+    }
+
+    public void databaseCreateContainerAsyncSample() {
+        String containerId = "passengers";
+        PartitionKeyDefinition partitionKeyDefinition = new PartitionKeyDefinition();
+        // BEGIN: com.azure.cosmos.CosmosAsyncDatabase.createContainer
+        CosmosContainerProperties containerProperties =
+            new CosmosContainerProperties(containerId, partitionKeyDefinition);
+        cosmosAsyncDatabase.createContainer(containerProperties)
+            .subscribe(
+                cosmosContainerResponse -> System.out.println(cosmosContainerResponse),
+                throwable -> System.out.printf("Container with id : %s already exists \n", containerId)
```

Comment:
error could happen because of any reason, we should change the error message to generic message, like -> 

```suggestion
                throwable -> System.out.printf("Failed to create container: " + throwable)
```

Reply:
Please change at other places as well.

Reply:
Thanks for comment. I've fixed error messages